src/GraphQLParser/Visitors/SDLWriter.cs

```diff
@@ -681,6 +697,7 @@ public override async ValueTask VisitArgument(GraphQLArgument argument, TContext
         public override async ValueTask VisitArgumentsDefinition(GraphQLArgumentsDefinition argumentsDefinition, TContext context)
         {
             await context.Write("(").ConfigureAwait(false);
+            await context.WriteLine().ConfigureAwait(false);
```

Comment:
?? This isn't typical.  At least, most graphql formatting tools, like 'prettify' in GraphiQL, would not do this

Reply:
https://github.com/graphql-dotnet/parser/pull/160/files/b1f15e0490ece23f5d820497450d809509c11857#r778103802